dumper/dataWritter.go

```diff
@@ -352,8 +352,9 @@ func generateInsertStatement(values []rowDataStructure, table schemareader.Table
 	columnNames := strings.Join(table.Columns, ", ")
 	valueFiltered := filterRowData(values, table)
 
-	if strings.Compare(table.MainUniqueIndexName, schemareader.VirtualIndexName) == 0 {
+	if strings.Compare(table.MainUniqueIndexName, schemareader.VirtualIndexName) == 0 || utils.Contains(OnlyIfParentExistsTables, table.Name) {
```

Comment:
We should not be using a channel specific field in this file, which should be generic. 
We need to think in another way to control this mechanism.

Reply:
done